src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = (outerArgument) => {
+    return (innerArgument) => {
+      count++;
+
+      if (count > 3 && count % 2 === 0) {
+        return `Bzzz... Error!`;
+      }
+
+      return outerArgument + innerArgument;
+    };
+  };
+
+  return getSum;
```

Comment:
Can also be returned right away.
```suggestion
  return (outerArgument) => {
    return (innerArgument) => {
      count++;

      if (count > 3 && count % 2 === 0) {
        return `Bzzz... Error!`;
      }

      return outerArgument + innerArgument;
    };
  };
```